src/Views/batchprocessor.html

```diff
@@ -15,7 +15,25 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
     <div style="padding-top: 10px;">
 
         <div id="submitBatch" ng-if="vm.selectedBatchProcessorTabName==='submitBatch'">
-            <p>This is the submit batch processor tab</p>
+            <div id="selectRegionID">
+                <label>Select State / Region:</label>
+
+                <select ng-model="selectedRegion" ng-options="region.name for region in vm.regionList | orderBy: 'name'" value="region.code"></select>

```

Comment:
```suggestion
                                <select ng-model="selectedRegion" ng-options="region.code as region.name for region in vm.regionList track by region.code"></select>

```

Your change was showing these value="object:XX" values rather than the code: 
![image](https://user-images.githubusercontent.com/40237491/218186577-e481d0d1-d319-44b5-bf0f-74c5c0efcde9.png)

The suggested change is one we used in the SCRunoff branch: https://github.com/USGS-WiM/StreamStats/blob/SC-Runoff/src/Views/scstormrunoff.html#L107 (the SCRunoff branch has several forms and may be helpful for future inspiration!)